sycl/test-e2e/Graph/event_profiling_info.cpp

```diff
@@ -0,0 +1,191 @@
+// REQUIRES: cuda || level_zero, gpu
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out 2>&1
+// RUN: %if ext_oneapi_level_zero %{env ZE_DEBUG=4 %{run} %t.out 2>&1 | FileCheck %s %}
+//
+// CHECK-NOT: LEAK
+
+// This test checks the profiling of an event returned
+// from graph submission with event::get_profiling_info().
+// It first tests a graph made exclusively of memory operations,
+// then tests a graph made of kernels.
+// The second run is to check that there are no leaks reported with the embedded
+// ZE_DEBUG=4 testing capability.
+
+#include "./graph_common.hpp"
+#include <unistd.h>
+
+#define VERBOSE 0
+
+#if VERBOSE
```

Comment:
We might want to rename this macro to make it more descriptive. E.g. `VERBOSE_PRINT` or even `GRAPH_TESTS_VERBOSE_PRINT`. Also do we need a numerical value here? or can we just set it to undefined by default like:

```suggestion
//#define GRAPH_TESTS_VERBOSE_PRINT

#ifdef GRAPH_TESTS_VERBOSE_PRINT
```

Reply:
I changed the name for a more explicit name : GRAPH_TESTS_VERBOSE_PRINT.
However, I kept the numerical value to avoid having commented lines in our test (according to Ewan's comment previously).